packages/autorest.typescript/test/rlcIntegration/generated/mediaTypesRest/samples-dev/analyzeBodyNoAcceptHeaderSample.ts

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+import createMediaTypesClient, {
+  AnalyzeBodyNoAcceptHeaderParameters
+} from "@msinternal/media-types-service-rest";
+import * as dotenv from "dotenv";
+
+dotenv.config();
+
+/**
+ * This sample demonstrates how to call operation AnalyzeBodyNoAcceptHeader
+ *
+ * @summary call operation AnalyzeBodyNoAcceptHeader
+ */
+async function analyzeBodyNoAcceptHeaderSample() {
+  const client = createMediaTypesClient();
+  const options: AnalyzeBodyNoAcceptHeaderParameters = {
+    body: "{Your body}",
```

Comment:
Is it possible to have some mock value or even empty value for the payload of the body?

Reply:
I saw some cases below that shows more details of the body, just curious why body properties are not shown here.

Reply:
```typescript
  body?:
    | string
    | Uint8Array
    | ReadableStream<Uint8Array>
    | NodeJS.ReadableStream;
```